### DISCUSSION TITLE
Google adwords form tracking

### DISCUSSION THREAD

**Discussion by MarkHuyton** [elementor/elementor#19868] [Developers Community]
I have been given some code by google to track when someone clicks on an google ad and then fills in a Elementor pro form, not sure what the [emailVariable] is, is this the field id which in this case is [field id="email"] Instructions from Google below

<script>
var ecData = {
"email": '[emailVariable]',
};
window.dataLayer = window.dataLayer || [];
window.dataLayer.push({
'event': 'formSubmit',
'ecData': ecData
});

</script>

// Please replace [emailVariable] with the variable that tracks the email from the form and add the customized code on the back end of the thank you page//

**Comment by hitsgee** [comment]
You can achieve this using the custom code section in the pro version. You can use the code snippets plugin if you are using only free.

**Reply by MarkHuyton** [reply]
Hi

The problem is not inserting the code but what to replace the emailVariable with
